ci/run-benchmark.yaml

```diff
@@ -0,0 +1,55 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: theorf/google-cloud-sdk-helm
+inputs:
+  - name: eq-survey-runner-benchmark
+params:
```

Comment:
I think it would be a good idea to list the params that this task uses here. This is highlighted as a common pattern at https://concourse-ci.org/task-step.html: 
> Looking into the task.yml, a common pattern is to list the param in the task definition with no value. This indicates that you expect the pipeline to provide the value.

Reply:
Ok this makes good sense to me - even if just to document what params are expected by the task so that it is easier to `fly execute`. One question - what is your opinion on having the params default to empty strings (such as `PROJECT_ID:` versus a default variable expansion such as `PROJECT_ID: ((google.project_id))`? If the former then anytime the param isn't passed to the task, it will become an empty string. I'm assuming here that variable expansion will work in that part of the task file.

Reply:
I think empty is best for now while we work through externalising all the tasks.